src/sdl2_compat.c

```diff
@@ -2709,6 +2709,35 @@ SDL_CreateRenderer(SDL_Window *window, int index, Uint32 flags)
     return SDL3_CreateRenderer(window, name, flags);
 }
 
+DECLSPEC int SDLCALL
+SDL_AudioInit(const char *driver_name)
+{
+    if (driver_name) {
+        SDL_SetHint("SDL_AUDIODRIVER", driver_name);
+    }
+    SDL_InitSubSystem(SDL_INIT_AUDIO);
+}
+
+DECLSPEC void SDLCALL
+SDL_AudioQuit(void)
+{
+    SDL_QuitSubSystem(SDL_INIT_AUDIO);
+}
+
+DECLSPEC int SDLCALL
+SDL_VideoInit(const char *driver_name)
+{
+    if (driver_name) {
+        SDL_SetHint("SDL_VIDEODRIVER", driver_name);
```

Comment:
SDL3 renamed `SDL_VIDEODRIVER` and `SDL_VIDEODRIVER` to `SDL_AUDIO_DRIVER` and `SDL_VIDEODRIVER`.
How does sdl2-compat handle this?

Reply:
here's a PR for this : 
https://github.com/libsdl-org/sdl2-compat/pull/14